src/admin/content-block/content-block.types.ts

```diff
@@ -98,15 +99,16 @@ export enum ContentBlockBackgroundColor {
 export enum ContentBlockType {
 	Buttons = 'BUTTONS',
 	Heading = 'HEADING',
+	CTAs = 'CTAS',
 	Intro = 'INTRO',
 	RichText = 'RICH_TEXT',
 	RichTextTwoColumns = 'RICH_TEXT_TWO_COLUMNS',
 }
 
 export interface HeadingBlockComponentState {
-	title: string;
-	level: HeadingLevelOptions;
-	align: AlignOptions;
+	children: string;
+	type: HeadingLevelOptions;
+	// align: AlignOptions;
```

Comment:
waarom is dit gecomment?

Reply:
Content Block is er nog niet op voorbereid. Heb al deze kleine zaken in een lijstje voor straks nog in 1x te fixen.